sky/utils/controller_utils.py

```diff
@@ -337,7 +337,7 @@ def get_controller_resources(
         controller_resources)[0]
 
     controller_exist = (global_user_state.get_cluster_from_name(
-        controller.value.name) is not None)
+        controller.value.cluster_name) is not None)
```

Comment:
When controller exists, why return controller_resources_to_use (which is copied from default resources) rather than that controller’s launched resources?

Reply:
I suppose the default controller resources can always be scheduled on existing controller, but changed to the existing resources for clarity.